sim/src/edge.rs

```diff
@@ -20,6 +20,7 @@ pub struct Edge {
     delay: u64,
     id: String,
     neighbors: Vec<String>,
+    seed: Option<u64>,
```

Comment:
Just generate a seed if it is not provided, then you do not have to use Option

Reply:
Good idea;  it is done.